worker/protos/ffmpeg_worker.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+service FFmpeg {
+  rpc transcode(Request) returns (stream Log) {}
+}
+
+message Log {
```

Comment:
It would be good if we return a Response message with a "status" field (which can be an enum and be populated from subprocess exit status ) along with the log.